release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -635,7 +642,11 @@ submodule openconfig-bgp-common {
           "Multipath parameters for eBGP";
         container link-bandwidth-ext-community {
           description
-            "Usage of DMZ Link-Bandwidth extended community";
+            "Usage of DMZ Link-Bandwidth extended community in FIB's
```

Comment:
Why the FIB reference here? This is both in the RIB and subsequent forwarding.

Reply:
I also am not sure what "in FIBs" means?  Can you provide a more detailed explanation of what is expected?  Is it not covered in the reference IETF draft?

Reply:
OK. I try to empfasis that this is NOT about BGP RIBs like Local-RIB or Adj-RIB-out. In BGP RIBs LBw stays as is, unaffected by this config. Hence this config has no impact on how LBw is handled on sessions.

Probably I can say "nexthops and nexthop groups programing in cross-protocol, network-instance scoped, global RIB". If this souds better?

Reply:
@dplore
This controlls system local behaviour which is by definition out of scope of IETF standaridation (standard track RFC and I.D.). Some of system internal behaviours are sometiomes published as informational RFC, but they are not standards and do not instatntiate any obligation across implementations.
This configuration do not impact interoperability between systems.